notebook_runtime.ipynb

```diff
@@ -122,20 +83,263 @@
     "        datetime.date(2020, 5, 10): 0.35,\n",
     "        datetime.date(2020, 5, 18): r0\n",
     "    }\n",
-    "]\n",
-    "\n",
-    "Dataset = CovidDatasets()\n",
-    "for state in Dataset.get_all_states_by_country('USA'):\n",
-    "    for i in range(0, len(INTERVENTIONS)):\n",
-    "        intervention = INTERVENTIONS[i]\n",
-    "        record_results(\n",
-    "            model_state('USA', state, intervention),\n",
-    "            'results/test',\n",
-    "            state,\n",
-    "            i,\n",
-    "            Dataset.get_population_by_country_state('USA', state)\n",
-    "        )\n"
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
```

Comment:
I suggest that we clear all cells prior to committing. This will make diffs easier to read. 

Unless the intent is to check in computed data.